### PR TITLE
Gen3 added

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: data/local.yaml [hunk 1]
```diff
@@ -1089,3 +1089,12 @@ S21F:
 S22F:
   description: Time-based milestone marking the completion of the Spring 2022 development
     cycle.
+LDM-GEN3:
```
Comment (gpdf) [inline data/local.yaml:1092]: Should this be LDM-503-Gen3 ?

File: data/local.yaml [hunk 2]
```diff
@@ -1089,3 +1089,11 @@ S21F:
...
+  description: 'Show parity of Butler Gen3 with Butler Gen2 systems.
+   We should be able to run processing pipelines with Gen3 which previously
+   ran on Gen2.
+   Demonstrate the ability to write new pipelines using the framework.
+   Framework should be ready to support Preops processing.'
```
Comment (gpdf) [inline data/local.yaml:1097]: Is it possible to clarify what "Preops processing" is, by referencing a pipelines milestone, or a document or requirement?

File: data/local.yaml [hunk 3]
```diff
@@ -1089,3 +1089,11 @@ S21F:
 S22F:
   description: Time-based milestone marking the completion of the Spring 2022 development
     cycle.
+LDM-GEN3:
+  description: 'Show parity of Butler Gen3 with Butler Gen2 systems.
+   We should be able to run processing pipelines with Gen3 which previously
```
Comment (gpdf) [inline data/local.yaml:1094]: I'd suggest a stronger phrasing, like "Demonstrate the ability to run under Gen3 key DRP and AP pipelines that previously ran on Gen2."

File: data/local.yaml (hunk 1 above)
Comment (womullan) [reply data/local.yaml:1092]: yes    but This is what is in P6 and rather than change it I will take it ..

File: data/local.yaml (hunk 3 above)
Comment (womullan) [reply data/local.yaml:1094]: I think its ok

File: data/local.yaml (hunk 1 above)
Comment (gpdf) [reply data/local.yaml:1092]: Oh, it's already in there.  OK, unfortunate.  I thought it was a new one.

**Pull request merged**